update_vendored_collections/update_vendored_collections.py

```diff
@@ -87,13 +87,28 @@ def spec_replace_sources(content, collection_tarballs):
     return content
 
 
+def spec_replace_bundled_provides(content, collection_tarballs):
+    print("Replacing bundled provides in the spec file")
+    for collection, tarball in collection_tarballs.items():
+        version = re.sub(r"\.tar\.gz", "", re.sub(".*-", "", tarball))
+        content = re.sub(
+            r"bundled\(ansible-collectionbundled\(ansible-collection\("
+            + collection
+            + r"\)\) = .*",
+            "bundled(ansible-collection(" + collection + ")) = " + version,
+            content,
+            flags=re.MULTILINE,
```

Comment:
Can you give me an example of the string you are trying to match?  I'm not sure what `bundled\(ansible-collectionbundled\(ansible-collection\(" + collection + r"\)\) = .*"` is supposed to match

Reply:
`Provides: bundled(ansible-collection(ansible.posix)) = 1.4.0`